src/python/national_voter_file/tests/test_transformers.py

```diff
@@ -17,6 +17,11 @@
 
 TEST_STATES = ['co', 'de', 'fl', 'mi', 'nc', 'ny', 'oh', 'ok', 'ut', 'wa']
 
+STATE_CODE_MAP = {'Colorado' : 'CO', 'Delaware' : 'DE', 'Florida' : 'FL',
```

Comment:
we can probably just have one dictionary, call it TEST_STATES, and then use TEST_STATES.values to get a list of the state initials? if not that python method, there's another one :)